### DISCUSSION TITLE
How to edit all layers on the shared vertex

### DISCUSSION THREAD

**Discussion by akifmertturk** [geoman-io/leaflet-geoman#1431] [Questions & Help] [answered]
I want to edit a vertex not the layer itself only. So what i want is that when i edit a ploygon's vertex, and drag/drop to some location, if there are other layers sharing the same location with the edited vertex, then their vertices also should updated with the new location. But for now, i could not find any solution for this, and i can't handle it. 
When i move the vertex of the polygon, other polygon's vertex is staying its position, and the neighborhood is broken in between poylgons.
Below, First image shows before edit. Second one is the currently edit mode when moving one vertex of the polygon. And the last one is what i want in deed.
Thanks.

<img width="538" alt="image" src="https://github.com/geoman-io/leaflet-geoman/assets/36456722/9cdff8c2-b347-4c1a-a47e-e8d9e733bf54">
<img width="538" alt="image" src="https://github.com/geoman-io/leaflet-geoman/assets/36456722/cf409b64-7f7f-498c-91dc-8bf3a5b2f933">
<img width="538" alt="image" src="https://github.com/geoman-io/leaflet-geoman/assets/36456722/1767c497-65a1-47bf-8d12-c95a656b3b98">

**Comment by Falke-Design** [comment, marked as answer]
Hi @akifmertturk,

we already implemented this feature in the Pro-Version: https://www.geoman.io/docs/options/pinning
I hope you can understand, that we can't help you with this topic in the open-source version.

![](https://user-images.githubusercontent.com/2399810/65375984-288ece80-dc9b-11e9-930e-bca03ad7cb56.gif)

Thank you.

**Comment by akifmertturk** [comment]
Thanks for your fast response!
